pymaker/auctions.py

```diff
@@ -660,3 +641,269 @@ def parse_event(self, event):
 
     def __repr__(self):
         return f"Flopper('{self.address}')"
+
+
+class Clipper(AuctionContract):
+    """A client for the `Clipper` contract, used to interact with collateral auctions.
+
+    You can find the source code of the `Clipper` contract here:
+    <https://github.com/makerdao/dss/blob/master/src/clip.sol>.
+
+    Attributes:
+        web3: An instance of `Web` from `web3.py`.
+        address: Ethereum address of the `Clipper` contract.
+    """
+
+    abi = Contract._load_abi(__name__, 'abi/Clipper.abi')
+    bin = Contract._load_bin(__name__, 'abi/Clipper.bin')
+
+    class KickLog:
```

Comment:
Is it worth having these extend `LogNote`?

Reply:
`flip` uses [LibNote](https://github.com/makerdao/dss/blob/1.2.10/src/flip.sol#L47), which has some relationship to `DS-Note`.  `clip` [does not appear to use](https://github.com/makerdao/dss/blob/liq-2.0/src/clip.sol#L49) this "note"-related stuff.  It also doesn't appear to be applied consistently through other contracts.  Specifically, `kick`s tend to be [emitted events](https://github.com/makerdao/dss/blob/1.2.10/src/flip.sol#L136) while [bids use 'note's](https://github.com/makerdao/dss/blob/1.2.10/src/flip.sol#L138).

Reply:
I could make a new baseclass called `EventLog` (assuming it won't shadow anything in Web3).  Haven't put much thought into it, but happy to discuss.

Reply:
Did a brief search of Web3, and it looks like it wouldn't conflict, but this is likely out of scope for this PR